### PR TITLE
п

### PR DESCRIPTION
**Pull request by HappyPala4**

### CODE REVIEW

**Comment by triplezeta** [comment]
🤨

File: donat/_donations.dm [hunk 1]
```diff
@@ -0,0 +1,167 @@
+/datum/donate_info
```
Comment (ZephyrTFA) [inline donat/_donations.dm:1]: You honest to god put micro transactions into your downstream? Why?

Comment (Latentish) [reply donat/_donations.dm:1]: Brother, what?

Comment (HappyPala4) [reply donat/_donations.dm:1]: > You honest to god put micro transactions into your downstream? Why?

I accidentally made this pull request to the repository

**Comment by HappyPala4** [comment]
> 🤨

I accidentally made this pull request to the repository

File: donat/_donations.dm (hunk 1 above)
Comment (Latentish) [reply donat/_donations.dm:1]: That doesn't answer the question.

**Pull request closed without merging**